packages/beacon-push-draw-id/src/handler.ts

```diff
@@ -0,0 +1,35 @@
+import { Relayer } from 'defender-relay-client';
+import { DefenderRelayProvider, DefenderRelaySigner } from 'defender-relay-client/lib/ethers';
+import { getContracts, beaconPushDrawId } from '@pooltogether/v4-autotasks-library';
+import { mainnet, testnet } from '@pooltogether/v4-pool-data';
+
+export async function handler(event: any) {
+  const provider = new DefenderRelayProvider(event);
+  const signer = new DefenderRelaySigner(event, provider, { speed: 'fast' });
+  const relayer = new Relayer(event);
+
+  const chainId = Number(process.env.CHAIN_ID);
+  const contracts = getContracts(chainId, mainnet, testnet);
```

Comment:
Originally, I've implemented `getContracts` in each autotasks cause they don't share the same `v4-pool-data`.
That being said, we can refactor and move the function to the library and pass the mainnet and testnet json to the function.
In this PR, I don't see the function in the library though.

Reply:
I just forked this setup from the existing autotasks in this repo. 

I'm not really sure what `getContracts` is supposed to do, I don't see an implementation anywhere. Is it meant to just switch on the `chainId` provided and return `mainnet` or `testnet` accordingly?